docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  app:
+    build: .
+    image: mxchip-gifs
+    container_name: mxchip-gifs
+    command: node server.js
+    ports:
+      - 3000:3000
+    volumes:
+      - ./:/usr/app
+      - /usr/app/node_modules
```

Comment:
I think I'm totally missing something, but where does this volume map to on a non unix system?